clerk/middleware.go

```diff
@@ -30,3 +31,21 @@ func addSessionToContext(request *http.Request, session *Session) *http.Request
 	updated := context.WithValue(ctx, ActiveSession, session)
 	return request.WithContext(updated)
 }
+
+func WithNetworkless(client Client) func(handler http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			token := r.Header.Get("Authorization")
+
+			claims, err := client.VerifyToken(token)
+			if err != nil {
+				w.WriteHeader(http.StatusUnauthorized)
```

Comment:
Is the status always going to be `401 Unauthorized`  for every error returned by the `VerifyToken`?